Makefile

```diff
@@ -1,3 +1,5 @@
+SHELL := bash
```

Comment:
This doesn't seem to be necessary.

Reply:
That's fair. I'd copied it from `master/Makefile`, where I didn't see any reason to prefer `bash` over `/bin/sh` there, either, just thinking that it's what people expected.

I also like it better without. Removing.

Thanks!